### ISSUE TITLE
Y3H Basal Auto-Activation Removal

### ISSUE THREAD

**Issue by Yydtqqqg** [lionking0000/recYnH#1]
Hi! I am using this published pipeline to process a sequencing dataset generated by rec-Y3H. After running `recYnH.py score`, we have a matrix where rows are RNAs (bait) and columns are proteins (prey). Now, do we need to remove auto-activators from the prey proteins? If so, I believe we should find the 75th percentile value for each column instead of each row, right? And this means we should slightly modify `recYnH.py merge` as it finds the 75th percentile for each row before running this script, right?

**Comment by lionking0000** [comment]
Hi, this auto-activator correction step will be automatically conducted during the merging step. But, it is an arbitrary filter so I would recommend testing auto-activation before the whole screening and removing them for the screening, which will increase a lot of screening sensitivity.

**Comment by Yydtqqqg** [comment]
You are right, better to do pre-screening. When I read the paper, I found this

> By cloning the X76 library only into the BD-containing pBWH vector and conducting an AD-empty screen (Fig. [2b](https://www.nature.com/articles/s41467-018-06128-x#Fig2)), we were able to detect which proteins auto-activate the Y2H reporters. In contrast, by cloning the X76 library only into the AD-containing pAWH vector and screening against the empty pBWH vector (Supplementary Fig. [8a](https://www.nature.com/articles/s41467-018-06128-x#MOESM1)), we were able to detect which proteins are toxic.

It seems to me only bait proteins can contain auto-activators and only prey proteins can contain very toxic proteins in Y2H system (bait proteins' toxicity is usually not high). I wonder if proteins (bait) in Y3H system can act as auto-activators. If so, without pre-testing auto-activation, what would you suggest to remove their effect?
